cockpit/build.gradle

```diff
@@ -39,15 +40,14 @@ android {
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
-    api "org.yaml:snakeyaml:1.20"
-
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    api 'org.yaml:snakeyaml:1.20'
     api "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     api "com.android.support:appcompat-v7:$supportLibraryVersion"
     api "com.android.support:recyclerview-v7:$supportLibraryVersion"
     api "com.android.support.constraint:constraint-layout:$constraintLayoutVersion"
-
     testImplementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     testImplementation "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
-    testImplementation "io.mockk:mockk:1.7.15"
+    testImplementation 'io.mockk:mockk:1.7.15'
+    implementation files("../CockpitCore/build/libs/CockpitCore-1.1.1-SNAPSHOT.jar")
```

Comment:
what about extracting core version to some variable and use `$core_version` here?